### PR TITLE
Update the Getting-Started page for Radio button, Check box, Radial menu and Switch controls

### PR DESCRIPTION
**Pull request by SivakumarVairamuthu**
### Description ###

Update the Getting-Started page for Radio button, Check box, Radial menu and Switch controls

### CODE REVIEW

File: MAUI/Radial-Menu/Getting-Started.md [hunk 1]
```diff
@@ -9,19 +9,33 @@ documentation: ug
 
 # Getting Started with .NET MAUI RadialMenu Control (SfRadialMenu)
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Radial-Menu/Getting-Started.md:10]: Getting Started with .NET MAUI Radial Menu

File: MAUI/Radial-Menu/Getting-Started.md [hunk 2]
```diff
@@ -9,19 +9,33 @@ documentation: ug
...
 
-This section provides a quick overview of how to get started with the [.NET MAUI RadialMenu control (SfRadialMenu)](https://www.syncfusion.com/maui-controls/maui-radial-menu) for .NET MAUI and a walk-through to configure the .NET MAUI RadialMenu control in a real-time scenario.
+This section guides you through setting up and configuring a [.NET MAUI RadialMenu control](https://www.syncfusion.com/maui-controls/maui-radial-menu) in your .NET MAUI application. Follow the steps below to add a basic [.NET MAUI RadialMenu control](https://www.syncfusion.com/maui-controls/maui-radial-menu) to your project.
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Radial-Menu/Getting-Started.md:12]: This section guides you through setting up and configuring a [.NET MAUI RadialMenu] (https://www.syncfusion.com/maui-controls/maui-radial-menu) in your .NET MAUI application. Follow the steps below to add a basic Radial Menu to your project.

File: MAUI/Radial-Menu/Getting-Started.md [hunk 3]
```diff
@@ -9,19 +9,33 @@ documentation: ug
...
 
-## Creating an Application using the .NET MAUI RadialMenu Control
 
-1. Create a new .NET MAUI application in Visual Studio.
 
-2. Syncfusion .NET MAUI components are available in [nuget.org](https://www.nuget.org/). To add the [.NET MAUI RadialMenu]() to your project, open the NuGet package manager in Visual Studio, search for `Syncfusion.Maui.RadialMenu`, and then install it.
+## Prerequisites
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Radial-Menu/Getting-Started.md:16]: ## Prerequisites

Before proceeding, ensure the following are set up:
1. Install [.NET 7 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/7.0) or later is installed.
1. Set up a .NET MAUI environment with Visual Studio 2022 (v17.3 or later) or VS Code. For VS Code users, ensure that the .NET MAUI workload is installed and configured as described [here.](https://learn.microsoft.com/en-us/dotnet/maui/get-started/installation?view=net-maui-8.0&tabs=visual-studio-code)

File: MAUI/Radial-Menu/Getting-Started.md [hunk 4]
```diff
@@ -9,19 +9,33 @@ documentation: ug
...
+Before proceeding, ensure the following are in place:
+1. Install .NET 7 SDK or later.
 
-3. To initialize the control, import the control namespace `Syncfusion.Maui.RadialMenu` in XAML or C# code.
+2. Set up a .NET MAUI environment with Visual Studio 2022 (v17.3 or later) or VS Code. For VS Code users, ensure that the .NET MAUI workload is installed and configured as per the instructions provided [here](https://learn.microsoft.com/en-us/dotnet/maui/get-started/installation?view=net-maui-8.0&tabs=visual-studio-code).
 
-4. Initialize [SfRadialMenu](https://www.syncfusion.com/maui-controls/maui-radial-menu).
+To get start quickly with our [.NET MAUI RadialMenu control](https://www.syncfusion.com/maui-controls/maui-radial-menu), you can check the below video.
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Radial-Menu/Getting-Started.md:22]: To get start quickly with our .NET MAUI Radial Menu, you can check the below video.

File: MAUI/CheckBox/Getting-Started.md [hunk 5]
```diff
@@ -9,21 +9,32 @@ documentation: ug
...
+This section guides you through setting up and configuring a [CheckBox](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Buttons.SfCheckBox.html) in your .NET MAUI application. Follow the steps below to add a basic CheckBox to your project.
 
-## Creating an application with .NET MAUI CheckBox
+## Prerequisites
 
-This section explains the steps required to work with the [CheckBox](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Buttons.SfCheckBox.html) control for .NET MAUI.
+Before proceeding, ensure the following are in place:
+
+    1. Install .NET 7 SDK or later.
+    2. Set up a .NET MAUI environment with Visual Studio 2022 (v17.3 or later) or VS Code. For VS Code users, ensure that the .NET MAUI workload  is installed and configured as described here.
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/CheckBox/Getting-Started.md:19]: 1. Set up a .NET MAUI environment with Visual Studio 2022 (v17.3 or later) or VS Code. For VS Code users, ensure that the .NET MAUI workload is installed and configured as described [here.](https://learn.microsoft.com/en-us/dotnet/maui/get-started/installation?view=net-maui-8.0&tabs=visual-studio-code)

File: MAUI/CheckBox/Getting-Started.md [hunk 6]
```diff
@@ -9,21 +9,32 @@ documentation: ug
...
-This section explains the steps required to work with the [.NET MAUI CheckBox](https://www.syncfusion.com/maui-controls/maui-checkbox) control.
+This section guides you through setting up and configuring a [CheckBox](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Buttons.SfCheckBox.html) in your .NET MAUI application. Follow the steps below to add a basic CheckBox to your project.
 
-## Creating an application with .NET MAUI CheckBox
+## Prerequisites
 
-This section explains the steps required to work with the [CheckBox](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Buttons.SfCheckBox.html) control for .NET MAUI.
+Before proceeding, ensure the following are in place:
+
+    1. Install .NET 7 SDK or later.
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/CheckBox/Getting-Started.md:18]: 1. Install [.NET 7 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/7.0) or later is installed.

File: MAUI/CheckBox/Getting-Started.md [hunk 7]
```diff
@@ -9,21 +9,32 @@ documentation: ug
...
 
 To get start quickly with our .NET MAUI CheckBox, you can check the below video.
 
 {% youtube "https://www.youtube.com/watch?v=dgekGX8eYMo" %}
 
-## Adding a .NET MAUI CheckBox reference
+## Step 1: Create a New MAUI Project
+
+    1. Launch Visual Studio or VS Code.
+    2. Navigate to File > New > Project and select the .NET MAUI App template.
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/CheckBox/Getting-Started.md:28]: Navigate to **File > New > Project,** then select the **.NET MAUI App** template.

File: MAUI/CheckBox/Getting-Started.md [hunk 8]
```diff
@@ -9,21 +9,32 @@ documentation: ug
...
+    3. Name the project and choose a location, then click Create.
+
+## Step 2: Install the Syncfusion MAUI Buttons NuGet Package
 
-Syncfusion .NET MAUI controls are available on [Nuget.org](https://www.nuget.org/). To add the [.NET MAUI CheckBox](https://www.syncfusion.com/maui-controls/maui-checkbox) to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.Maui.Buttons](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Buttons.html), and install it.
+    1. In Solution Explorer, right-click the project and choose Manage NuGet Packages.
+    2. Search for Syncfusion.Maui.Buttons and install the latest version.
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/CheckBox/Getting-Started.md:34]: Search for `Syncfusion.Maui.Buttons` and install the latest version.

File: MAUI/Radial-Menu/Getting-Started.md [hunk 9]
```diff
@@ -9,19 +9,33 @@ documentation: ug
...
 
-## Register the handler
+{% youtube "https://www.youtube.com/watch?v=VxufXFzSh0M" %}
+
+## Step 1: Create a New MAUI Project
+
+1. Open Visual Studio or VS Code.
+2. Go to File > New > Project and choose the .NET MAUI App template.
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Radial-Menu/Getting-Started.md:29]: Navigate to **File > New > Project,** then select the **.NET MAUI App** template.

File: MAUI/Radial-Menu/Getting-Started.md [hunk 10]
```diff
@@ -9,19 +9,33 @@ documentation: ug
...
+3. Name the project and choose a location, then click Create.
+
+## Step 2: Install the Syncfusion MAUI RadialMenu NuGet Package
+
+1. In Solution Explorer, right-click the project and choose Manage NuGet Packages.
+2. Search for [Syncfusion.Maui.RadialMenu](https://www.nuget.org/packages/Syncfusion.Maui.RadialMenu) and install the latest version.
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Radial-Menu/Getting-Started.md:35]: Search for `Syncfusion.Maui.RadialMenu` and install the latest version.

File: MAUI/CheckBox/Getting-Started.md [hunk 11]
```diff
@@ -9,23 +9,34 @@ documentation: ug
...
+
+
+## Step 1: Create a New MAUI Project
+
+1. Launch Visual Studio or VS Code.
+1. Navigate to **File > New > Project,** then select the **.NET MAUI App** template.
+1. Name the project and choose a location, then click Create.
 
-Syncfusion .NET MAUI controls are available on [Nuget.org](https://www.nuget.org/). To add the [.NET MAUI CheckBox](https://www.syncfusion.com/maui-controls/maui-checkbox) to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.Maui.Buttons](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Buttons.html), and install it.
+## Step 2: Install the Syncfusion MAUI Core NuGet Package
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/CheckBox/Getting-Started.md:31]: Install the Syncfusion MAUI Button NuGet Package

File: MAUI/CheckBox/Getting-Started.md [hunk 12]
```diff
@@ -58,69 +69,46 @@ namespace ButtonSample
...
-	             x:Class="GettingStarted.MainPage">
-        <ContentPage.Content>
-            <StackLayout>
-                  <syncfusion:SfCheckBox x:Name="checkBox"/>
-            </StackLayout>
-        </ContentPage.Content>
-    </ContentPage>
+<ContentPage
+    . . .    
+    xmlns:sfavatar="clr-namespace:Syncfusion.Maui.Buttons;assembly=Syncfusion.Maui.Buttons">
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/CheckBox/Getting-Started.md:84]: xmlns:buttons="clr-namespace:Syncfusion.Maui.Buttons;assembly=Syncfusion.Maui.Buttons">

File: MAUI/CheckBox/Getting-Started.md [hunk 13]
```diff
@@ -221,6 +209,8 @@ The check box can be used as a single or as a group. A single check box mostly u
 
 ![.NET MAUI CheckBox](Images/Getting-Started/termsandconditions.png)
 
+## Using Multiple CheckBoxes
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/CheckBox/Getting-Started.md:212]: Multiple CheckBoxes

File: MAUI/Radial-Menu/Getting-Started.md [hunk 14]
```diff
@@ -7,23 +7,37 @@ control: RadialMenu (SfRadialMenu)
...
-## Register the handler
 
-The [Syncfusion.Maui.Core](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.html) NuGet is a dependent package for all Syncfusion controls of .NET MAUI. In the `MauiProgram.cs` file, register the handler for Syncfusion core.
+## Step 1: Create a New MAUI Project
+
+1. Launch Visual Studio or VS Code.
+1. Navigate to **File > New > Project,** then select the **.NET MAUI App** template.
+1. Name the project and choose a location, then click Create.
+
+## Step 2: Install the Syncfusion MAUI Core NuGet Package
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Radial-Menu/Getting-Started.md:32]: Install the Syncfusion MAUI RadialMenu NuGet Package

File: MAUI/Radial-Menu/Getting-Started.md [hunk 15]
```diff
@@ -50,95 +64,71 @@ The [Syncfusion.Maui.Core](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.C
...
-                InitializeComponent();
-                StackLayout stackLayout = new StackLayout();
-                SfRadialMenu radialMenu = new SfRadialMenu();
-                stackLayout.Children.Add(syncfusion);
-                this.Content = radialMenu;
-            }
+            InitializeComponent();           
+            StackLayout stackLayout = new StackLayout();
+            SfRadialMenu radialMenu = new SfRadialMenu();
+            stackLayout.Children.Add(syncfusion);
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Radial-Menu/Getting-Started.md:98]: stackLayout.Children.Add(radialMenu);

File: MAUI/Radio-Button/Getting-Started.md [hunk 16]
```diff
@@ -8,21 +8,37 @@ documentation: ug
...
 
-In the MauiProgram.cs file, register the handler for the Syncfusion core.
+
+## Step 1: Create a New MAUI Project
+
+1. Launch Visual Studio or VS Code.
+1. Navigate to **File > New > Project,** then select the **.NET MAUI App** template.
+1. Name the project and choose a location, then click Create.
+
+## Step 2: Install the Syncfusion MAUI Core NuGet Package
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Radio-Button/Getting-Started.md:33]: Install the Syncfusion MAUI Buttons NuGet Package

File: MAUI/Radio-Button/Getting-Started.md [hunk 17]
```diff
@@ -55,61 +71,37 @@ namespace ButtonSample
...
 {% endhighlight %}
 
-## Create a Simple .NET MAUI SfRadioButton
+## Step 4: Add a Basic RadioButton
 
-Step 1: Add the NuGet to the project as discussed in the above reference section. 
-
-Step 2: Add the namespace as shown in the following code sample, to initialize the .NET MAUI Radio Button.
+ To initialize the control, import the Core namespace into your code.
+1. Initialize [SfAvatarView](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.SfAvatarView.html?tabs=tabid-1).
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Radio-Button/Getting-Started.md:77]: Initialize [SfRadioButton](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Buttons.SfRadioButton.html).

File: MAUI/Radio-Button/Getting-Started.md [hunk 18]
```diff
@@ -55,61 +71,37 @@ namespace ButtonSample
...
-	                x:Class="GettingStarted.MainPage">
-        <ContentPage.Content>
-            <StackLayout>
-                <syncfusion:SfRadioButton x:Name="radioButton"/>        
-            </StackLayout>
-        </ContentPage.Content>
-     </ContentPage>
+<ContentPage
+    . . .    
+    xmlns:sfavatar="clr-namespace:Syncfusion.Maui.Buttons;assembly=Syncfusion.Maui.Buttons">
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Radio-Button/Getting-Started.md:85]: xmlns:syncfusion="clr-namespace:Syncfusion.Maui.Buttons;assembly=Syncfusion.Maui.Buttons">

File: MAUI/Radio-Button/Getting-Started.md [hunk 19]
```diff
@@ -145,24 +137,27 @@ The two different visual states of the [.NET MAUI Radio Button](https://help.syn
 
 To change the state of the .NET MAUI Radio Button, you can utilize the [IsChecked](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Buttons.SfRadioButton.html#Syncfusion_Maui_Buttons_SfRadioButton_IsChecked) property of [SfRadioButton](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Buttons.SfRadioButton.html). When the Radio Button is checked, an inner circle is added to its visualization.
 
-The .NET MAUI Radio Buttons are used when there is a list of two or more options or groups that are mutually exclusive, and the user must select exactly one choice, such as "Select Gender" or "Choose the best option!".
+
+## Grouping RadioButtons with RadioGroup
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Radio-Button/Getting-Started.md:141]: remove this title

File: MAUI/Switch/Getting-Started.md [hunk 20]
```diff
@@ -8,17 +8,36 @@ documentation: ug
...
+2. Set up a .NET MAUI environment with Visual Studio 2022 (v17.3 or later) or VS Code. For VS Code users, ensure that the .NET MAUI workload is installed and configured as described [here.](https://learn.microsoft.com/en-us/dotnet/maui/get-started/installation?view=net-maui-8.0&tabs=visual-studio-code)
+
+
+## Step 1: Create a New MAUI Project
+
+1. Launch Visual Studio or VS Code.
+1. Navigate to **File > New > Project,** then select the **.NET MAUI App** template.
+1. Name the project and choose a location, then click Create.
+
+## Step 2: Install the Syncfusion MAUI Core NuGet Package
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Switch/Getting-Started.md:32]: Install the Syncfusion MAUI Buttons NuGet Package

File: MAUI/Switch/Getting-Started.md [hunk 21]
```diff
@@ -50,67 +69,63 @@ namespace SwitchSample
...
 
 {% tabs %}
 
 {% highlight xaml %}
 
-<syncfusion:SfSwitch />
-	
+<ContentPage
+    . . .    
+    xmlns:sfavatar="clr-namespace:Syncfusion.Maui.Buttons;assembly=Syncfusion.Maui.Buttons">
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Switch/Getting-Started.md:83]: xmlns:syncfusion="clr-namespace:Syncfusion.Maui.Buttons;assembly=Syncfusion.Maui.Buttons">

File: MAUI/Switch/Getting-Started.md [hunk 22]
```diff
@@ -50,67 +69,63 @@ namespace SwitchSample
...
+    }
 
 {% endhighlight %}
 
 {% endtabs %}
 
 ![SfSwitch](images/getting-started/SfSwitch.png)
 
-## Performing an action based on state
+## Performing an action based on switch state
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Switch/Getting-Started.md:111]: revert this changes

File: MAUI/Radial-Menu/Getting-Started.md (hunk 3 above)
Comment (SivakumarVairamuthu) [reply MAUI/Radial-Menu/Getting-Started.md:18]: Updated the changes

File: MAUI/CheckBox/Getting-Started.md [hunk 23]
```diff
@@ -9,23 +9,34 @@ documentation: ug
...
 
-## Adding a .NET MAUI CheckBox reference
+## Prerequisites
+
+Before proceeding, ensure the following are set up:
+1. Install [.NET 7 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/7.0) or later is installed.
+2. Set up a .NET MAUI environment with Visual Studio 2022 (v17.3 or later) or VS Code. For VS Code users, ensure that the .NET MAUI workload is installed and configured as described [here.](https://learn.microsoft.com/en-us/dotnet/maui/get-started/installation?view=net-maui-8.0&tabs=visual-studio-code)
+
+
+## Step 1: Create a New MAUI Project
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/CheckBox/Getting-Started.md:25]: need to update this step

File: MAUI/CheckBox/Getting-Started.md [hunk 24]
```diff
@@ -9,23 +9,34 @@ documentation: ug
...
 
-## Handler registration 
+1. In **Solution Explorer,** right-click the project and choose **Manage NuGet Packages.**
+1. Search for [Syncfusion.Maui.Buttons](https://www.nuget.org/packages/Syncfusion.Maui.Buttons/) and install the latest version.
+1. Ensure the necessary dependencies are installed correctly, and the project is restored.
 
-In the MauiProgram.cs file, register the handler for the Syncfusion core.
+## Step 3: Register the handler
+
+Syncfusion.Maui.Core NuGet is a dependent package for all Syncfusion controls of .NET MAUI. In the MauiProgram.cs file, register the handler for Syncfusion core.
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/CheckBox/Getting-Started.md:39]: Add link for NuGet

File: MAUI/Radial-Menu/Getting-Started.md [hunk 25]
```diff
@@ -7,23 +7,37 @@ control: RadialMenu (SfRadialMenu)
...
+Before proceeding, ensure the following are set up:
+1. Install [.NET 7 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/7.0) or later is installed.
+2. Set up a .NET MAUI environment with Visual Studio 2022 (v17.3 or later) or VS Code. For VS Code users, ensure that the .NET MAUI workload is installed and configured as described [here.](https://learn.microsoft.com/en-us/dotnet/maui/get-started/installation?view=net-maui-8.0&tabs=visual-studio-code)
 
-4. Initialize [SfRadialMenu](https://www.syncfusion.com/maui-controls/maui-radial-menu).
 
-## Register the handler
 
-The [Syncfusion.Maui.Core](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.html) NuGet is a dependent package for all Syncfusion controls of .NET MAUI. In the `MauiProgram.cs` file, register the handler for Syncfusion core.
+## Step 1: Create a New MAUI Project
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Radial-Menu/Getting-Started.md:26]: Need to update this step

File: MAUI/Radial-Menu/Getting-Started.md [hunk 26]
```diff
@@ -7,23 +7,37 @@ control: RadialMenu (SfRadialMenu)
...
+
+## Step 2: Install the Syncfusion MAUI RadialMenu NuGet Package
+
+1. In **Solution Explorer,** right-click the project and choose **Manage NuGet Packages.**
+1. Search for [Syncfusion.Maui.RadialMenu](https://www.nuget.org/packages/Syncfusion.Maui.RadialMenu/) and install the latest version.
+1. Ensure the necessary dependencies are installed correctly, and the project is restored.
+
+## Step 3: Register the handler
+
+Syncfusion.Maui.Core NuGet is a dependent package for all Syncfusion controls of .NET MAUI. In the MauiProgram.cs file, register the handler for Syncfusion core.
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Radial-Menu/Getting-Started.md:40]: Add NuGet link

File: MAUI/Switch/Getting-Started.md [hunk 27]
```diff
@@ -94,23 +110,22 @@ this.Content = sfSwitch;
...
-<syncfusion:SfSwitch StateChanged="SfSwitch_StateChanged"/>
+<syncfusion:SfSwitch x:Name="sfSwitch" StateChanged="SfSwitch_StateChanged"/>
 	
 {% endhighlight %}
 
 {% highlight C# %}
 
 SfSwitch sfSwitch = new SfSwitch();
 sfSwitch.StateChanged+= SfSwitch_StateChanged;
-this.Content = sfSwitch;
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Switch/Getting-Started.md:113]: revert this changes

**Review by naveenkumar-sanjeevirayan — APPROVED** [review]
